### ISSUE TITLE
[REQUEST] Allow users to select WCAG compliance level

### ISSUE THREAD

**Issue by aaronreed708** [finos/a11y-theme-builder#347]
## Suggestion/Concern

@lwnoble suggested today that we add a switch in Theme Builder to allow a user to designate whether the design they are creating is for targeting WCAG AA or AAA compliance.  The sdk would ensure that the generated design system would be properly compliant for its designated spec.

WCAG AAA compliance  creates applications with wider contrast differentials and minimum target areas, for example, but designs build along those lines are generally less pleasing to the eye.

## Proposed Solution

Lise suggested that the user should be allowed to choose which version they are targeting on the "Create a new design system" dialog.
